source/index.html

```diff
@@ -1,12 +1,100 @@
 <!DOCTYPE html>
 <html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Кэт энерджи</title>
-  </head>
-  <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS, уровень 2</a>».</p>
-
-  </body>
+<head>
+  <meta charset="UTF-8">
+  <title>HTML Academy: Кэт энерджи</title>
+</head>
+<body>
+<!--Эта часть повторяется еще в form и catalog, копировать туда изменения -->
+<header class="page-header">
+  <div class="page-header__div">
+    <a class="cat-logo" href="index.html"><img class="cat-logo__image" src="#" alt="Логотип компании Кэт Энерджи."></a>
+    <a class="logo" href="index.html"><img class="logo__image" src="#" alt="Текстовый логотип Кэт Энерджи."></a>
+    <button class="page-header__button-close button-close" type="button" aria-label="Закрыть"></button>
+  </div>
+  <nav class="site-navigation">
+    <ul class="site-navigation__list site-nav-list">
+      <li class="site-nav-list__item"><a class="site-nav-list__link" href="catalog.html">Каталог продукции</a></li>
+      <li class="site-nav-list__item"><a class="site-nav-list__link" href="index.html">Главная</a></li>
+      <li class="site-nav-list__item"><a class="site-nav-list__link" href="form.html">Подбор программы</a></li>
+    </ul>
+  </nav>
+</header>
+<main class="page-main">
+  <section class="product-promotion">
+    <h1 class="product-promotion__headline main-headline main-headline--white">Функциональное питание для котов</h1>
+    <p class="product-promotion__slogan">Занялся собой? Займись котом!</p>
+    <img class="product-promotion__image" src="#" alt="Коробка с питанием для котов и кошек Cat Energy Pro.">
+    <a class="product-promotion__link order-button" href="form.html">Подобрать программу</a>
+  </section>
+  <section class="weight">
+    <h2 class="weight__headline cat-weight-headline">Похудение</h2>
+    <p class="weight__promo">Ваш кот весит больше собаки и почти утратил способность лазить по деревьям? Пора на диету! Cat Energy Slim поможет вашему питомцу сбросить лишний вес.</p>
+    <a class="weight__link catalog-link-slim" href="catalog.html#500-slim">Каталог Slim</a>
+  </section>
+  <section class="weight">
+    <h2 class="weight__headline cat-weight-headline">Набор массы</h2>
+    <p class="weight__promo">Заработать авторитет среди дворовых котов и даже собак? Серия Cat Energy Pro поможет вашему коту нарастить необходимые мышцы!</p>
+    <a class="weight__link catalog-link-pro" href="catalog.html#500-pro">Каталог pro</a>
+  </section>
+  <section class="principles">
+    <h2 class="principles__headline index-section-headline">Как это работает</h2>
```

Comment:
не привязываемся к индексу, заголовки сквозные по всему сайту
```suggestion
    <h2 class="principles__headline section-headline">Как это работает</h2>
```